test/e2e/connectivity_test.go

```diff
@@ -31,12 +31,53 @@ import (
 
 const pingCount = 5
 
+// TestConnectivity is the top-level test which contains all subtests for
+// Connectivity related test cases so they can share setup, teardown.
+func TestConnectivity(t *testing.T) {
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	t.Run("testPodConnectivityOnSameNode", func(t *testing.T) {
+		testPodConnectivityOnSameNode(t, data)
+	})
+	t.Run("testHostPortPodConnectivity", func(t *testing.T) {
+		skipIfHasWindowsNodes(t)
+		testHostPortPodConnectivity(t, data)
+	})
+	t.Run("testPodConnectivityDifferentNodes", func(t *testing.T) {
+		skipIfNumNodesLessThan(t, 2)
+		testPodConnectivityDifferentNodes(t, data)
+	})
+	t.Run("testPodConnectivityAfterAntreaRestart", func(t *testing.T) {
+		skipIfHasWindowsNodes(t)
+		testPodConnectivityAfterAntreaRestart(t, data)
+	})
+	t.Run("testOVSRestartSameNode", func(t *testing.T) {
+		skipIfProviderIs(t, "kind", "test not valid for the netdev datapath type")
+		skipIfNotIPv4Cluster(t)
+		skipIfHasWindowsNodes(t)
+		testOVSRestartSameNode(t, data)
+	})
+	t.Run("testOVSFlowReplay", func(t *testing.T) {
+		skipIfProviderIs(t, "kind", "stopping OVS daemons create connectivity issues")
+		skipIfHasWindowsNodes(t)
+		testOVSFlowReplay(t, data)
+	})
+	t.Run("testPingLargeMTU", func(t *testing.T) {
+		skipIfNumNodesLessThan(t, 2)
+		testPingLargeMTU(t, data)
+	})
+}
+
 func waitForPodIPs(t *testing.T, data *TestData, podInfos []podInfo) map[string]*PodIPs {
 	t.Logf("Waiting for Pods to be ready and retrieving IPs")
 	podIPs := make(map[string]*PodIPs)
 	for _, pi := range podInfos {
 		podName := pi.name
-		if podIP, err := data.podWaitForIPs(defaultTimeout, podName, testNamespace); err != nil {
+		if podIP, err := data.podWaitForIPs(120*time.Second, podName, testNamespace); err != nil {
```

Comment:
why is the change?

Reply:
last time there was a pod waiting timeout issue in IPv6 only e2e cases, we thought it's  caused by big image, so increase this, but it turns out it's not the case, the original issue have been fixed, I will revert this back to defaultTimeout.

Reply:
done